ui/src/tasks/components/TaskRunsList.tsx

```diff
@@ -4,51 +4,134 @@ import React, {PureComponent} from 'react'
+      'finishedAt',
+      'status',
+      'scheduledFor',
+    ]
     return (
-      <IndexList>
-        <IndexList.Header>
-          <IndexList.HeaderCell columnName="Requested At" width="20%" />
-          <IndexList.HeaderCell columnName="Started At" width="20%" />
-          <IndexList.HeaderCell columnName="Finished At" width="20%" />
-          <IndexList.HeaderCell columnName="Status" width="10%" />
-          <IndexList.HeaderCell columnName="Schedule For" width="20%" />
-          <IndexList.HeaderCell columnName="" width="10%" />
-        </IndexList.Header>
-        <IndexList.Body
-          emptyState={
-            <EmptyState size={ComponentSize.Large}>
-              <EmptyState.Text
-                text={"Looks like this Task doesn't have any Runs"}
-                highlightWords={['Runs']}
-              />
-            </EmptyState>
-          }
-          columnCount={6}
-        >
-          {this.listRuns}
-        </IndexList.Body>
-      </IndexList>
+      <>
+        <IndexList>
+          <IndexList.Header>
+            <IndexList.HeaderCell
+              columnName="Requested At"
+              width="20%"
+              sortKey={headerKeys[0]}
+              sort={sortKey === headerKeys[0] ? sortDirection : Sort.None}
+              onClick={this.handleClickColumn}
+            />
+            <IndexList.HeaderCell
+              columnName="Started At"
+              width="20%"
+              sortKey={headerKeys[1]}
+              sort={sortKey === headerKeys[1] ? sortDirection : Sort.None}
+              onClick={this.handleClickColumn}
+            />
+            <IndexList.HeaderCell
+              columnName="Finished At"
+              width="20%"
+              sortKey={headerKeys[2]}
+              sort={sortKey === headerKeys[2] ? sortDirection : Sort.None}
+              onClick={this.handleClickColumn}
+            />
+            <IndexList.HeaderCell
+              columnName="Status"
+              width="10%"
+              sortKey={headerKeys[3]}
+              sort={sortKey === headerKeys[3] ? sortDirection : Sort.None}
+              onClick={this.handleClickColumn}
+            />
+            <IndexList.HeaderCell
+              columnName="Schedule For"
+              width="20%"
+              sortKey={headerKeys[4]}
+              sort={sortKey === headerKeys[4] ? sortDirection : Sort.None}
+              onClick={this.handleClickColumn}
+            />
+            <IndexList.HeaderCell width="10%" />
+          </IndexList.Header>
+          <IndexList.Body
+            emptyState={
+              <EmptyState size={ComponentSize.Large}>
+                <EmptyState.Text
+                  text={"Looks like this Task doesn't have any Runs"}
+                  highlightWords={['Runs']}
+                />
+              </EmptyState>
+            }
+            columnCount={5}
+          >
+            {this.sortedRuns}
+          </IndexList.Body>
+        </IndexList>
+      </>
```

Comment:
I don't think you need the fragment here

Reply:
True, I will remove it